content-scripts/cs.js

```diff
@@ -465,7 +465,7 @@ const showBanner = () => {
   });
   const notifInnerText1 = Object.assign(document.createElement("span"), {
     style: NOTIF_TEXT_STYLE,
-    innerHTML: escapeHTML(chrome.i18n.getMessage("extensionUpdateInfo1_v1_17", DOLLARS)).replace(
+    innerHTML: escapeHTML(chrome.i18n.getMessage("extensionUpdateInfo1_v1_18", DOLLARS)).replace(
```

Comment:
If you just delete the old update notifications, why don't you get rid of the version in the name so you don't have to change this every update?

Reply:
Because of translation. If a string changes completely it makes no sense to use the same key.

Reply:
It was actually easier for me when the same key was used for each version because I could see my previous translations and translate things like "new addon" and "go to Scratch Addons settings to enable it" the same way.

Reply:
Yeah, as a translator I kinda agree but some other translator gave another reason why it should be a different key and at the time I considered it valid enough to change how I do this (I can't rememeber exactly now)

Reply:
Would it be possible to make the parts that are the same in every version separate strings and insert them into the string as parameters?

Reply:
It's possible